lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I noticed the m9k ak example did AddCSLuaFile. You might need to do that here as well so the changes are mirrored on the client.

Reply:
oops